### PR TITLE
Updated PyPi documentation

### PR DESCRIPTION
**Pull request by mryzhov**
Updated documentation for PyPi distribution
We need to describe its content and all possible packages

### CODE REVIEW

File: docs/install_guides/installing-openvino-pip.md [hunk 1]
```diff
@@ -22,51 +45,22 @@ python3 -m pip install --upgrade pip
...
-```sh
-python -c "import os, sys; print((os.path.dirname(sys.executable))+'\Library\\bin')"
-```
-- Virtual environments or custom Python installations (from sources or tarball):
-```sh
-python -c "import os, sys; print((os.path.dirname(sys.executable))+'\..\Library\\bin')"
-```
-
-### Step 4. Verify that the package is installed
+### Step 4. Verify that the Runtime package is installed
```
Comment (ryanloney) [inline docs/install_guides/installing-openvino-pip.md:59]: This should be Step 3 not Step 4.

File: docs/install_guides/installing-openvino-pip-dev.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+### Step 2. Install the Intel® distribution of OpenVINO™ toolkit
+
+Run the command below: <br>
+
+   ```sh
+   pip install openvino-dev
+   ```
+
+### Step 4. Verify that the Runtime package is installed
```
Comment (ryanloney) [inline docs/install_guides/installing-openvino-pip-dev.md:47]: This should be Step 3 not Step 4.

File: docs/install_guides/installing-openvino-pip-dev.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+|-----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Model Optimizer](../MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md) | This tool imports, converts, and optimizes models that were trained in popular frameworks to a format usable by Intel tools, especially the Inference Engine. <br>Popular frameworks include Caffe\*, TensorFlow\*, MXNet\*, and ONNX\*.                                                                              |
+| Additional Tools                                   | A set of tools to work with your models including [Accuracy Checker utility](https://docs.openvinotoolkit.org/latest/omz_tools_accuracy_checker_README.html), [Post-Training Optimization Tool](https://docs.openvinotoolkit.org/latest/pot_README.html)  |
+
```
Comment (ryanloney) [inline docs/install_guides/installing-openvino-pip-dev.md:18]: Mention Inference Engine so developers are aware that the runtime is included.

File: docs/install_guides/installing-openvino-pip-dev.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+## System Requirements
+
+* [Python* distribution](https://www.python.org/) 3.6, 3.7, 3.8
+* Operating Systems:
```
Comment (ryanloney) [inline docs/install_guides/installing-openvino-pip-dev.md:22]: Change to "Supported Operating Systems"

File: docs/install_guides/installing-openvino-pip-dev.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  - Ubuntu* 18.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
+  - Ubuntu* 20.04 long-term support (LTS), 64-bit (python 3.8)
+  - macOS* 10.15.x versions
+  - Windows 10*, 64-bit Pro, Enterprise or Education (1607 Anniversary Update, Build 14393 or higher) editions
+  - Windows Server* 2016 or higher
+
```
Comment (ryanloney) [inline docs/install_guides/installing-openvino-pip-dev.md:28]: Add this: "**NOTE:** This package can be installed on many versions of Linux, but only the specific versions above are fully validated."

File: docs/install_guides/installing-openvino-pip.md [hunk 6]
```diff
@@ -1,17 +1,40 @@
...
 
-* [Python* distribution](https://www.python.org/) 3.6 or 3.7
+* [Python* distribution](https://www.python.org/) 3.6, 3.7, 3.8
 * Operating Systems:
-  - Ubuntu* 18.04 long-term support (LTS), 64-bit
+  - Ubuntu* 18.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
+  - Ubuntu* 20.04 long-term support (LTS), 64-bit (python 3.8)
   - macOS* 10.15.x versions
   - Windows 10*, 64-bit Pro, Enterprise or Education (1607 Anniversary Update, Build 14393 or higher) editions
   - Windows Server* 2016 or higher
```
Comment (ryanloney) [inline docs/install_guides/installing-openvino-pip.md:32]: Add this: "**NOTE:** This package can be installed on many versions of Linux, but only the specific versions above are fully validated."

File: docs/install_guides/installing-openvino-pip.md [hunk 7]
```diff
@@ -1,17 +1,40 @@
...
+|-----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Inference Engine](../IE_DG/inference_engine_intro.md)               | This is the engine that runs the deep learning model. It includes a set of libraries for an easy inference integration into your applications.                                                                                                                                                                |
 
 ## System Requirements
 
-* [Python* distribution](https://www.python.org/) 3.6 or 3.7
+* [Python* distribution](https://www.python.org/) 3.6, 3.7, 3.8
 * Operating Systems:
```
Comment (ryanloney) [inline docs/install_guides/installing-openvino-pip.md:26]: Change this to "Supported Operating Systems"

File: docs/install_guides/installing-openvino-pip-dev.md (hunk 3 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip-dev.md:17]: Added section "Installed by dependency"

File: docs/install_guides/installing-openvino-pip-dev.md (hunk 4 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip-dev.md:28]: fixed

File: docs/install_guides/installing-openvino-pip.md (hunk 7 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip.md:25]: Changed

File: docs/install_guides/installing-openvino-pip.md (hunk 6 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip.md:32]: Added

Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip.md:32]: Added

File: docs/install_guides/installing-openvino-pip.md (hunk 7 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip.md:25]: Done

File: docs/install_guides/installing-openvino-pip-dev.md (hunk 5 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip-dev.md]: Added

File: docs/install_guides/installing-openvino-pip-dev.md (hunk 4 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip-dev.md:28]: Done

File: docs/install_guides/installing-openvino-pip-dev.md (hunk 2 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip-dev.md]: Fixed

File: docs/install_guides/installing-openvino-pip.md (hunk 1 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip.md]: Fixed

File: docs/install_guides/installing-openvino-pip-dev.md (hunk 3 above)
Comment (mryzhov) [reply docs/install_guides/installing-openvino-pip-dev.md:17]: Added "Installed by dependency" paragraph, please review one more time

File: docs/install_guides/pypi-openvino-dev.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+license terms for third party or open source software included in or with the Software Package, and your use indicates your acceptance of all such terms. Please refer to the “third-party-programs.txt” or other similarly-named text file included with the Software Package for additional details.
+
+## Introduction
+
+OpenVINO™ toolkit is a comprehensive toolkit for quickly developing applications and solutions that solve a variety of tasks including emulation of human vision, automatic speech recognition, natural language processing, recommendation systems, and many others. Based on latest generations of artificial neural networks, including Convolutional Neural Networks (CNNs), recurrent and attention-based networks, the toolkit extends computer vision and non-vision workloads across Intel® hardware, maximizing performance. It accelerates applications with high-performance, AI and deep learning inference deployed from edge to cloud.
+
+**Included with the Installation and installed by default:**
```
Comment (andrew-zaytsev) [inline docs/install_guides/pypi-openvino-dev.md:11]: We need to emphasize this is the developers package. For example:

The developers package includes the following components installed by default:

File: docs/install_guides/pypi-openvino-dev.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+> NOTE: This package can be installed on many versions of Linux, but only the specific versions above are fully validated.
+
+## Install the runtime or developer packages using the PyPI repository
+
+### Step 1. Set up and update pip to the highest version
+
+Run the command below:
+```sh
+python3 -m pip install --upgrade pip
+```
```
Comment (andrew-zaytsev) [inline docs/install_guides/pypi-openvino-dev.md:43]: What's the result of running this command? How we user can see that the package is successfully installed? Any messages on this?

File: docs/install_guides/pypi-openvino-dev.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+   ```sh
+   pip install openvino-dev
+   ```
+
+### Step 3. Verify that the package is installed
+
+Run the command below:
+```sh
+python3 -c "pot -h"
```
Comment (andrew-zaytsev) [inline docs/install_guides/pypi-openvino-dev.md:57]: What's the result of running this command? How we user can see that the package is successfully installed? Any messages on this?

File: docs/install_guides/pypi-openvino-rt.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+## Introduction
+
+OpenVINO™ toolkit is a comprehensive toolkit for quickly developing applications and solutions that solve a variety of tasks including emulation of human vision, automatic speech recognition, natural language processing, recommendation systems, and many others. Based on latest generations of artificial neural networks, including Convolutional Neural Networks (CNNs), recurrent and attention-based networks, the toolkit extends computer vision and non-vision workloads across Intel® hardware, maximizing performance. It accelerates applications with high-performance, AI and deep learning inference deployed from edge to cloud.
+
+The Intel® Distribution of OpenVINO™ toolkit for Linux\*:
+- Enables CNN-based deep learning inference on the edge
+- Supports heterogeneous execution across Intel® CPU, Intel® Integrated Graphics, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs
+- Speeds time-to-market via an easy-to-use library of computer vision functions and pre-optimized kernels
+
+**Included with the Installation and installed by default:**
```
Comment (andrew-zaytsev) [inline docs/install_guides/pypi-openvino-rt.md:16]: We need to highlight this is the runtime package. For example:

The runtime package includes the following components installed by default:

File: docs/install_guides/pypi-openvino-rt.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
+# Install Intel® Distribution of OpenVINO™ Toolkit Runtime Package
```
Comment (andrew-zaytsev) [inline docs/install_guides/pypi-openvino-rt.md:1]: Install Intel® Distribution of OpenVINO™ Toolkit Runtime Package --> Intel® Distribution of OpenVINO™ Toolkit Runtime Package

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: docs/install_guides/pypi-openvino-dev.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+|-----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Inference Engine](https://pypi.org/project/openvino)               | This is the engine that runs the deep learning model. It includes a set of libraries for an easy inference integration into your applications.                                                                                                                                                                |
+
+
+## Supported Operating Systems
```
Comment (andrew-zaytsev) [inline docs/install_guides/pypi-openvino-dev.md:25]: I guess, @ryanloney suggested changing not this line from `System Requirements` to `Supported Operating Systems`, but the line below: `Operating Systems` to `Supported Operating Systems`. Is it right?

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: docs/install_guides/pypi-openvino-dev.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+* Operating Systems:
+  - Ubuntu* 18.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
+  - Ubuntu* 20.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
+  - macOS* 10.15.x versions
+  - Windows 10*, 64-bit Pro, Enterprise or Education (1607 Anniversary Update, Build 14393 or higher) editions
+  - Windows Server* 2016 or higher
+> NOTE: This package can be installed on many versions of Linux, but only the specific versions above are fully validated.
+
+## Install the runtime or developer packages using the PyPI repository
+
```
Comment (ryanloney) [inline docs/install_guides/pypi-openvino-dev.md:37]: Add Step 0. 
"It is highly recommended that you use a virtual environment. 
Create and activate virtual environment:
```
python3 -m pip install --user virtualenv
virtualenv openvino_env
source openvino_env/bin/activate
```

File: docs/install_guides/pypi-openvino-dev.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
+# Intel® Distribution of OpenVINO™ Toolkit Developer Package 
+
+> **LEGAL NOTICE**: Your use of this software and any required dependent software (the
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md:3]: Yes, this message came from legal instructions, now we are waiting for the final approval

File: docs/install_guides/pypi-openvino-dev.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+## Supported Operating Systems
+
+* [Python* distribution](https://www.python.org/) 3.6, 3.7, 3.8
+* Operating Systems:
+  - Ubuntu* 18.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
+  - Ubuntu* 20.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
+  - macOS* 10.15.x versions
+  - Windows 10*, 64-bit Pro, Enterprise or Education (1607 Anniversary Update, Build 14393 or higher) editions
+  - Windows Server* 2016 or higher
+> NOTE: This package can be installed on many versions of Linux, but only the specific versions above are fully validated.
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md:34]: Agree, it is better to mention all the systems

File: docs/install_guides/pypi-openvino-dev.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+|-----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Inference Engine](https://pypi.org/project/openvino)               | This is the engine that runs the deep learning model. It includes a set of libraries for an easy inference integration into your applications.                                                                                                                                                                |
+
+
+## Supported Operating Systems
+
+* [Python* distribution](https://www.python.org/) 3.6, 3.7, 3.8
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md:27]: Because we do not support all python versions

[the remark this answers is not in the document]
Comment (andrew-zaytsev) [reply docs/install_guides/pypi-openvino-dev.md:27]: But it's not a supported operating system )

[the remark this answers is not in the document]
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md:27]: reverted system requirements

File: docs/install_guides/pypi-openvino-dev.md [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+* Operating Systems:
+  - Ubuntu* 18.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
+  - Ubuntu* 20.04 long-term support (LTS), 64-bit (python 3.6 or 3.7)
+  - macOS* 10.15.x versions
+  - Windows 10*, 64-bit Pro, Enterprise or Education (1607 Anniversary Update, Build 14393 or higher) editions
+  - Windows Server* 2016 or higher
+> NOTE: This package can be installed on many versions of Linux, but only the specific versions above are fully validated.
+
+## Install the runtime or developer packages using the PyPI repository
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md]: fixed

File: docs/install_guides/pypi-openvino-dev.md (hunk 8 above)
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md]: done

File: docs/install_guides/pypi-openvino-dev.md (hunk 10 above)
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md:78]: Added expected result

File: docs/install_guides/pypi-openvino-rt.md (hunk 12 above)
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-rt.md]: Changed

File: docs/install_guides/pypi-openvino-dev.md (hunk 13 above)
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md]: Yes, my bad, fixed

File: docs/install_guides/pypi-openvino-dev.md (hunk 14 above)
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md:37]: A bit modified and added, please review

File: docs/install_guides/pypi-openvino-dev.md [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+
+### Step 1. Set up and update pip to the highest version
+
+Run the command below:
+```sh
+python3 -m pip install --upgrade pip
+```
+
+### Step 2. Install the Intel® distribution of OpenVINO™ toolkit
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md]: fixed

File: docs/install_guides/pypi-openvino-rt.md (hunk 11 above)
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-rt.md]: done

File: docs/install_guides/pypi-openvino-dev.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md]: changed

File: docs/install_guides/pypi-openvino-dev.md [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+
+Run the command below: <br>
+
+   ```sh
+   pip install openvino-dev
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply docs/install_guides/pypi-openvino-dev.md:71]: added expected results

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**